consensus/src/txn_hash_and_authenticator_deduper.rs

```diff
@@ -168,8 +179,28 @@ mod tests {
         seq_num: u64,
         gas_unit_price: u64,
     ) -> RawTransaction {
-        let payload = aptos_stdlib::aptos_coin_transfer(receiver, 1);
-        raw_txn(payload, sender, seq_num, gas_unit_price)
+        let entry_func = EntryFunction::new(
+            ModuleId::new(
+                AccountAddress::new([
```

Comment:
there's a constant for this AccountAddress::ONE

Reply:
Done.